### ISSUE TITLE
Literal string 'assign' and 'assume' policies should be re-implemented in troposphere

### ISSUE THREAD

**Issue by philvarner** [Miserlou/Zappa#1234]
The current base 'assign' and 'assume' policies are created as literal strings in the code.  These should be re-implemented as troposphere/awacs Template Policy Statements.  

## Context
This is a first step towards more modular permissions, as it allows for more programmatic manipulation of the permissions, instead of simply appending to a dictionary generated from JSON or YAML. 

## Expected/Actual Behavior
No expected behavior change.

**Comment by Miserlou** [comment]
Yeah, I think this is a good idea I've never liked having these big static strings in the code anyway. There are some legacy policies still hardcoded in there as well, I think.

We should have a chat about the larger direction of this as well.
